detections/endpoint/gsuite_drive_share_in_external_email.yml

```diff
@@ -0,0 +1,45 @@
+name: Gsuite Drive Share In External Email
+id: f6ee02d6-fea0-11eb-b2c2-acde48001122
+version: 1
+date: '2021-08-16'
+author: Teoderick Contreras, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: This search is to detect suspicious google drive or google docs files shared outside or externally. 
+  This behavior might be a good hunting query to monitor exfitration of data made by an attacker to a compromise machine.
+search: 'index=obs-gsuite sourcetype=gsuite:drive:json NOT(email IN ("*splunk.com", "null"))
+  | stats values(parameters.doc_title) as doc_title count min(_time) as firstTime max(_time) as lastTime by 
```

Comment:
@tccontre is this a part of the Google suite where someone clicks a share button in Google Doc/Presentation/Spreadsheet? If so, this would be reasonable informational detection - otherwise, we'd "scream" on any email with attachment.

Either way, let's try to boost this one. This is our experience, a lot of times companies have a lot of external collaboration that would trigger a lot on this. For example, Splunk used to share a lot of such docs with auditors and consulting compaines (PwC, etc.). Other companies would have even more of this.

Thus, let's extract a domain out of email, and do a simple event stats command on a domain field - idea: remove these common external domains (barring gmail, yahoo mail, mail.com, etc.).

Reply:
thanks @miskoSplunk for great points and tips. will take note of this one and improve this draft detection